ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json

```diff
@@ -4,14 +4,16 @@
 		"approvers": [
 			"Shilpa-Gokul",
 			"prb112",
+			"pkenchap",
 			"werled",
 			"multiarch-approvers"
 		],
 		"reviewers": [
 			"Shilpa-Gokul",
 			"prb112",
+			"pkenchap",
 			"werled",
 			"multiarch-reviewers"
 		]
 	}
-}
\ No newline at end of file
+}
```

Comment:
The metadata check is complaining about an extra return at the end. `diff -Naupr .//ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json /tmp/tmp.mFAw1KgdTt/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json`

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44651/pull-ci-openshift-release-master-step-registry-metadata/1715344464936964096

ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json

Reply:
Removed the new line added from the truncate command .